input_files/all_imt_all_agg_input.toml

```diff
@@ -0,0 +1,38 @@
+[general]
+compatibility_key = "NZSHM22-0"
+hazard_model_id = "DEMO_MODEL"
+
+[logic_trees]
+model_version = "NSHM_v1.0.4"
+#model_version = "NSHM_v1.0.0"
```

Comment:
Is it worth cleaning up the commented lines?

Reply:
cleaned up commented lines